app/controllers/conference_registrations_controller.rb

```diff
@@ -27,8 +27,9 @@ def new
   end
 
   def show
-    @total_price = Ticket.total_price(@conference, current_user, paid: true)
+    @total_price = Ticket.total_price_user(@conference, current_user)
```

Comment:
where are you checking now that the tickets that you are taking into account are paid? I think you removed it, in this casem why? :confused:

Reply:
and you could use `@tickets` as a parameter, as that what you want to sum, right?

Reply:
Removing paid: true is a mistake.
I will use @tickets :smile:  thanks